include/soloud.h

```diff
@@ -37,7 +37,7 @@ freely, subject to the following restrictions:
 #endif
 
 #if !defined(DISABLE_SSE)
-#ifdef _MSC_VER
+#if defined(__x86_64__) || defined( _M_X64 ) || defined( __i386 ) || defined( _M_IX86 )
```

Comment:
Ahh nope, just checked, __i386 seems to be the one to be checking for
rather than __i386__.

On 28 May 2015 at 20:59, Neil Richardson <neilo@neilo.gd> wrote:

> On second thought the define may actually be __i386__ for detection of
> x86. I really dislike all the standard defines for this stuff!
>
> Mind, this is something that can be fixed when upgrading the genie scripts
> and just explicitely specify SSE support depending on the target :)
>
> On 28 May 2015 at 20:49, Jari Komppa <notifications@github.com> wrote:
>
>> In include/soloud.h
>> <https://github.com/jarikomppa/soloud/pull/104#discussion_r31221952>:
>>
>> > @@ -37,7 +37,7 @@ freely, subject to the following restrictions:
>> >  #endif
>> >
>> >  #if !defined(DISABLE_SSE)
>> > -#ifdef _MSC_VER
>> > +#if defined(__x86_64__) || defined( _M_X64 ) || defined( __i386 ) || defined( _M_IX86 )
>>
>> Not sure about this.. __i386? Also, does msvc set one (or more) of those?
>> Generally speaking I suppose just detecting intel architecture is fine as
>> everything out there supports at least SSE2 nowadays, but to be pedantic,
>> __i386 covers a lot of old stuff that doesn't.. =)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jarikomppa/soloud/pull/104/files#r31221952>.
>>
>
>